CMakeLists.txt

```diff
@@ -780,8 +780,14 @@ option(WANT_SPELLCHECKER "Compile spell-checker support" ON)
 if(WANT_SPELLCHECKER)
 	find_package(PkgConfig)
 	pkg_check_modules(Enchant enchant)
-	if(Enchant_FOUND)
-		set(CMAKE_STATUS_SPELLCHECKER_SUPPORT "Yes")
+	pkg_check_modules(Enchant2 enchant-2)
```

Comment:
Why not `pkg_search_module(Enchant enchant enchant-2)` instead of this whole change?

Reply:
Good idea! I thought we'd need to do some source modifications, but it doesn't seem to be the case.